_posts/2024-03-19-credential-stores-on-client-side.adoc

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: 'Using Credential Stores on for WildFly Client'
```

Comment:
typo in title `on for`

Reply:
fixed all comments till here. 
Thanks @Skyllarr and @luck3y